### DISCUSSION TITLE
How to install Klaviyo?

### DISCUSSION THREAD

**Discussion by jaequery** [vercel/next.js#61639] [App Router]
Anyone know what is the best way to integrate Nextjs with Klaviyo using only front-end (no backend)? Googling results in all kinds of old documents and the new one isn't very clear about this.

**Comment by Ricardo-Caceres** [comment]
You can use the JavaScript API, Klaviyo documentation speaks about onsite metric, that help me to put in Next.js:

https://developers.klaviyo.com/en/docs/guide_to_integrating_a_platform_without_a_pre_built_klaviyo_integration

And the API Reference explain about how use APIs for each case:

https://developers.klaviyo.com/en/reference/api_overview
